symengine/lib/symengine_wrapper.pyx

```diff
@@ -2714,6 +2732,36 @@ def module_cleanup():
 import atexit
 atexit.register(module_cleanup)
 
+def add_expand(a):
+    cdef Basic _a = sympify(a)
+    require(_a, Add)
+    cdef RCP[const symengine.Add] a1 = symengine.rcp_static_cast_Add(_a.thisptr)
+    return c2py(symengine.add_expand(a1))
+
+expand_add = add_expand
+
+def mul_expand(a):
+    cdef Basic _a = sympify(a)
+    require(_a, Mul)
+    cdef RCP[const symengine.Mul] a1 = symengine.rcp_static_cast_Mul(_a.thisptr)
+    return c2py(symengine.mul_expand(a1))
+
+expand_mul = mul_expand
+
+def pow_expand(a):
+    cdef Basic _a = sympify(a)
+    require(_a, Pow)
+    cdef RCP[const symengine.Pow] a1 = symengine.rcp_static_cast_Pow(_a.thisptr)
+    return c2py(symengine.pow_expand(a1))
+
+expand_pow = pow_expand
```

Comment:
Besides, `pow_expand`, `mul_expand` and `add_expand` should be removed from the public API